pkg/sql/sem/builtins/builtins.go

```diff
@@ -1863,7 +1863,7 @@ CockroachDB supports the following flags:
 				return roundDecimal(&args[0].(*tree.DDecimal).Decimal, scale)
 			},
 			Info: "Keeps `decimal_accuracy` number of figures to the right of the zero position " +
-				" in `input using half away from zero rounding. If `decimal_accuracy` " +
+				" in `input` using half away from zero rounding. If `decimal_accuracy` " +
```

Comment:
nit: There's an extra space between `position` and `in`. You could either remove it from the beginning of this string of the previous one.